i18n-helpers/src/gettext.rs

```diff
@@ -108,6 +130,14 @@ mod tests {
         catalog
     }
 
+    fn create_book(items: Vec<BookItem>) -> Book {
+        let mut book = Book::new();
+        items.into_iter().for_each(|item| {
+            book.push_item(item);
+        });
```

Comment:
I think a normal for-loop should work here and I think that would be a little simpler:

```suggestion
        for item in items {
            book.push_item(item);
        }
```

Reply:
Yeah guess I was too eager on doing things "the rust way" :D